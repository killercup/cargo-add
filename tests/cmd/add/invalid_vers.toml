bin.name = "cargo-add"
args = ["add", "my-package", "--vers", "invalid version string"]
status.code = 1
fs.sandbox = true

[env.add]
CARGO_IS_TEST="1"
