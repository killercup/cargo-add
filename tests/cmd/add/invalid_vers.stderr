Command failed due to unhandled error: Invalid dependency version requirement

Caused by: unexpected character 'i' while parsing major version number
